src/evm-block-extractor/src/database/postgres_db_client.rs

```diff
@@ -169,6 +172,30 @@ impl DatabaseClient for PostgresDbClient {
         Ok(())
     }
 
+    async fn insert_certified_block_data(&self, response: CertifiedBlock) -> anyhow::Result<()> {
```

Comment:
Why do we need to keep all these certified blocks in the DB if the API always returns the last one?

Reply:
[discussed]
This was a decision made last week to periodically save certified blocks. In this case we will be able to add endpoints to get the nearest certified block.